eldritch/events.py

```diff
@@ -93,9 +93,18 @@ def log(self, state) -> str:
 
 
 class ChoiceEvent(Event):
+  def __init__(self):
+    super().__init__()
+    self._choices = None
+
+  @property
+  def choices(self):
```

Comment:
Every choice event that we have has `choices`, unsurprisingly.  But adding it into the base class shuts my IDE up when it warns me about using a ChoiceEvent's choices.

Reply:
I think I tried an attribute initially, but while most classes use it as an attribute, but CityMovement already has a property. I also don't remember exactly what the error was, but yeah, unintuitive.